File: solutions/3-strings/1168/1168.c
#include <stdio.h>

int main() {
    int l[] = {6, 2, 5, 5, 4, 5, 6, 3, 7, 6};
    int n; scanf("%d", &n);
    while (n--) {
        char v[102]; scanf("%s", v);
        int ans = 0;
        for (int i = 0; v[i]; ++i) {
            ans += l[v[i] - '0'];
        }
        printf("%d leds\n", ans);
    }
    return 0;
}

File: solutions/3-strings/1168/1168.py
l = [6, 2, 5, 5, 4, 5, 6, 3, 7, 6]

n = int(input())

for _ in range(n):
    v = input()
    ans = sum([l[int(c)] for c in v])
    print(f'{ans} leds')
